adminPage/adminPageActions.php

```diff
@@ -1,58 +1,27 @@
 <?php
-require '../dbConn.php';
-
-$projectsTitle = $_POST['portfolioTitle'];
-$projectsText = $_POST['projectText'];
-$projectsUrl = $_POST['projectUrl'];
-
-$projectsInsertQuery = $db->prepare("INSERT INTO `projects` (`title`, `fillText`, `url`) VALUES (:projectsTitle, :projectsText, :projectsUrl)");
-$projectsInsertQuery->bindParam(':projectsTitle',$projectsTitle);
-$projectsInsertQuery->bindParam(':projectsText',$projectsText);
-$projectsInsertQuery->bindParam(':projectsUrl',$projectsUrl);
-$projectsInsertQuery->execute();
-
-/*
- * Function that is designed to take in the db, select the projects table and fetch all its data. Being used to retrieve the data displayed on the admin page.
- *
- * @param - called $db, designed to take the pre-existing $db variable
- *
- * @return - an array - of all info in the projects table on the db
- */
-function retrieveProjectsData($db) {
-    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-    $projectsEditQuery = $db->prepare("SELECT * FROM `projects`;");
-    $projectsEditQuery->execute();
-    $projectsResult = $projectsEditQuery->fetchAll();
-    return $projectsResult;
+require_once '../dbConn.php';
+require_once '../functions.php';
+
+if($_POST['portfolioTitle'] && $_POST['projectText'] && $_POST['projectUrl']){
```

Comment:
PSR2 violation. 
If statements with multiple conditoin should be set on different lines like
if ( 
      condition1
     conditoin2
) {